docs/git-guide.md

```diff
@@ -1357,6 +1357,47 @@ HEAD is now at 99aa2bf Add provision.py fails issue in common erros
 Current branch review-1913 is up to date.
 ```
 
+### Fetch a pull request without rebasing
+
+`tools/fetch-pull-request` is a similar to `tools/fetch-rebase-pull-request`, but
+it does not rebase the pull request with any changes from upstream/master,
```

Comment:
Just noticed this. I copied this line from the docs for `fetch-rebase-pull-request` so I'll update both.